src/compile/scale.ts

```diff
@@ -66,11 +97,12 @@ export function type(fieldDef: FieldDef, channel: Channel, mark: Mark): string {
     case NOMINAL:
       return 'ordinal';
     case ORDINAL:
+      if (channel === COLOR) {
+        return 'linear'; // time has order, so use interpolated ordinal color scale.
```

Comment:
The comment here looks out of place.  

Better comment that this return scale type for one of a few scales (2 for ordinal, 3 for bin) that color ramp scale uses  + why should `scale.type` returns the linear one.